### ISSUE TITLE
macOS: support retina display

### ISSUE THREAD

**Issue by PhilippvK** [muesli/telephant#21]
When I am using the highDPI display setting on my Macbook, Icons and Profile pictures look blurry compared to the text and menus.

 
<img width="867" alt="Screenshot 2019-05-14 at 10 08 00" src="https://user-images.githubusercontent.com/7712605/57681997-20f65380-7631-11e9-8bd2-006583068587.png">

**Comment by CalmBit** [comment]
Is there any chance that converting these images to SVGs would help fix this?

**Comment by muesli** [comment]
Yeah, we should switch to using SVGs where it's reasonable. The graphics displayed in the message panes may have to remain PNG based though, as it's simply a lot faster to render. We can ship with higher resolution images for HiDPI displays though.

**Comment by wezm** [comment]
Just a note that this isn't just macOS as per the issue title. I see the same thing on Linux.
